source/isaaclab/isaaclab/utils/timer.py

```diff
@@ -60,36 +63,68 @@ class Timer(ContextDecorator):
     Reference: https://gist.github.com/sumeet/1123871
     """
 
-    timing_info: ClassVar[dict[str, float]] = dict()
+    timing_info: ClassVar[dict[str, dict[str, float]]] = dict()
     """Dictionary for storing the elapsed time per timer instances globally.
 
     This dictionary logs the timer information. The keys are the names given to the timer class
     at its initialization. If no :attr:`name` is passed to the constructor, no time
     is recorded in the dictionary.
     """
 
-    def __init__(self, msg: str | None = None, name: str | None = None):
+    enable = True
```

Comment:
Is this a class variable?

```suggestion
    enable: ClassVar[bool] = True
```

Reply:
Yes, I need to rename it. I'll do a pass now to address all the comments! Thanks :)